Sources/XCTApodini/Response+Checks.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  Response+Checks.swift
+//  
+//
+//  Created by Paul Schmiedmayer on 2/3/21.
+//
+
+#if DEBUG
+@testable import Apodini
+import NIO
+import XCTest
+
+extension Empty: Equatable {
+    public static func == (lhs: Self, rhs: Self) -> Bool {
+        true
+    }
+}
+
+/// Adds the possibility to easily check the of a `Handler` by investigating the `Response`
+/// - Parameters:
```

Comment:
I think there is a word missing :). Applies to all documentation in this file.

Reply:
Thanks a lot, I will fix this 👍